### PR TITLE
Initial Education and Sustainability Plans, #37

### PR DESCRIPTION
**Pull request by Maestro0202**

### CODE REVIEW

**Comment by lhao03** [comment]
Random capitalization and missing periods, are these sections meant to be bulleted lists? 
image

File: src/human-practices/education/lets-talk-science.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+To provide university level technical training
+
+## Plan
+Brainstorming and ideation of classroom activities
+Pitching idea to LTS representatives 
+Marketing and Branding
+Contacting highschools through Let’s Talk Science 
+Volunteer Training for our internal team
+Final activity preparations + material gathering
+Executing our program with classroom visits
```
Comment (lhao03) [inline src/human-practices/education/lets-talk-science.md:25]: To make this a list, add `-` before each point:

```
- point 1
- point 2
```

you will get 

- point 1
- point 2

**Review by lhao03 — CHANGES_REQUESTED** [review]

**Review by lhao03 — APPROVED** [review]

**Review by jxin04 — CHANGES_REQUESTED** [review]
if these are bullet points, remove periods at the end and remove capitals for words in the middle

File: src/human-practices/sustainability/sustainability.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- To establish a network of recurring and new guests who can contribute valuable insights to the podcast
+
+### Non-Goals
+- To venture into discussions that do not have a clear link to these areas. 
+- Increasing viewership and using it for funding purposes
+
+## Plan
+- Identify and prioritize SDGs that have not been covered in previous sessions or new advancements
+- Brainstorm SDGs that can also be project specific
+- Reach out to previous guests who have been on the podcast before as well as new voices and experts who can bring unique perspectives to the podcast.
```
Comment (lhao03) [inline src/human-practices/sustainability/sustainability.md:24]: remove period here

File: src/human-practices/sustainability/sustainability.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+## Goals and non-goals
+### Goals
+- To continue exploring the intersection of synthetic biology with the SDGs.
+- Enrich the audience’s understanding and appreciation for the field’s potential to impact global sustainability positively 
+- To cultivate meaning connections
+- To gain valuable relationships with influential Health Professionals
+- To establish a network of recurring and new guests who can contribute valuable insights to the podcast
+
+### Non-Goals
+- To venture into discussions that do not have a clear link to these areas.
```
Comment (lhao03) [inline src/human-practices/sustainability/sustainability.md:18]: remove period here

File: src/human-practices/education/lets-talk-science.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+- To inspire students to pursue further education and careers in the STEM fields, particularly in synthetic biology and related disciplines. 
+
+### Non-Goals
+- To target audiences outside of the highschool education level for this specific initiative
+- To provide university level technical training
+
+## Plan
+Brainstorming and ideation of classroom activities
+- Pitching idea to LTS representatives 
+- Marketing and Branding
```
Comment (lhao03) [inline src/human-practices/education/lets-talk-science.md:21]: lowercase the "b" in branding; these are just some issues I saw at a glance, please check if there are any more!

**Review by lhao03 — COMMENTED** [review]
Please double check all bullet points!

File: src/human-practices/sustainability/sustainability.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+## Context and Scope
+This initiative will build on the success of the previous two years that it has been run, aiming to deepen the conversation around the SDGs by connecting listeners with experts from diverse fields. The podcast will not only educate its audience on the potential of synthetic biology in addressing global challenges but also foster a community of like-minded individuals' passion about sustainable development. 
+
+## Goals and non-goals
+### Goals
+- To continue exploring the intersection of synthetic biology with the SDGs
+- Enrich the audience’s understanding and appreciation for the field’s potential to impact global sustainability positively 
+- To cultivate meaning connections
+- To gain valuable relationships with influential Health Professionals
```
Comment (lhao03) [inline src/human-practices/sustainability/sustainability.md:14]: Lowercase the "H" in health professionals

File: src/human-practices/sustainability/sustainability.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+### Non-Goals
+- To venture into discussions that do not have a clear link to these areas 
+- Increasing viewership and using it for funding purposes
+
+## Plan
+- Identify and prioritize SDGs that have not been covered in previous sessions or new advancements
+- Brainstorm SDGs that can also be project specific
+- Reach out to previous guests who have been on the podcast before as well as new voices and experts who can bring unique perspectives to the podcast.
+- Figure out how to enhance production quality (Better question, recording, editing, design, etc)
```
Comment (lhao03) [inline src/human-practices/sustainability/sustainability.md:25]: Lowercase "B" in better

**Review by lhao03 — CHANGES_REQUESTED** [review]

**Review by lhao03 — APPROVED** [review]
Thanks!

**Review by jxin04 — APPROVED** [review]

**Pull request merged**